### PR TITLE
[FEATURE] Added exception when product not active; cleanups

### PR DESCRIPTION
**Pull request by turbo-ele**
only return special prices of active products in ``findAllCurrent`` function

### CODE REVIEW

File: Product/Exception/ProductNotActiveException.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+{
+    /**
+     * The name of the entity.
+     *
+     * @var string
+     */
+    private $entityName;
+
+    /**
+     * The id of the object not found.
```
Comment (christianbader) [inline Product/Exception/ProductNotActiveException.php]: ?

**Comment by christianbader** [comment]
else ack

File: composer.json [hunk 2]
```diff
@@ -31,7 +31,10 @@
         "symfony-cmf/testing": "~1.2",
         "sulu/document-manager": "dev-develop",
         "sulu/test-bundle": "dev-develop",
-        "massive/search-bundle": "0.7.*"
+        "massive/search-bundle": "dev-develop",
+        "pulse00/ffmpeg-bundle": "~0.5.2",
+        "php-ffmpeg/php-ffmpeg": "0.5.*",
```
Comment (hacfi) [inline composer.json]: The two `ffmpeg` dependencies are not used at the moment.

Comment (danrot) [inline composer.json]: @turbo-ele Why have you added them here?

Comment (turbo-ele) [inline composer.json]: they 'have been" a dependency by sulu/sulu: "dev-develop"
"composer update" was not possible without that

Comment (danrot) [inline composer.json]: Are you sure that's not only because of the search bundle?

Comment (turbo-ele) [inline composer.json]: pretty sure, since composer told me that it cannot install since sulu/sulu needs ffmpeg to continue.

**Pull request merged**